vipps-login-from-phone-number-api-howitworks.md

```diff
@@ -14,44 +14,72 @@ END_METADATA -->
+If the scope is included in the request, the end user will be asked to give consents according to merchant specifications.
+
+|                           Delegated consents shown                            |                           Delegated consents approved                            |
+|:-----------------------------------------------------------------------------:|:--------------------------------------------------------------------------------:|
+| ![User sees delegated consents in Vipps](images/delegated-consents-step1.png) | ![User confirm delegated consents in Vipps](images/delegated-consents-step2.png) |
+
+Some of the elements on the delegatedConsents screen can be adjusted to fit the merchant's needs
+ * A top text where the purpose of collecting consents should be stated. This is customizable for you as a merchant.
+ * The consents wanted by the merchant (see more [When using delegatedConsents, which consents does Vipps support?](vipps-login-api-faq.md#when-using-delegatedconsents-which-consents-does-vipps-support)).
+   These can be individually specified to be mandatory or optional. Mandatory consents must be ticked before the user can complete the login flow. See what consents we currently support, or reach out if you need something else and we'll se what we can do!
+ * A bottom text where we provide some contextual information for the end user. The text is fixed, but the links will point to the merchant's terms and conditions, as well as privacy statement.
+
+### 4. Redirect or end the login in Vipps
+
+The merchant controls what happens after the user has confirmed the login and sharing of information.
 
-The merchant controls what happens after the user has  confirmed the login and sharing of information. The merchant can end the process in Vipps by giving confirmation of successful completion in Vipps, or they can have the user taken to a page in the browser on the phone.
+The merchant can end the login process in Vipps, where the user will receive a confirmation screen that the login is complete.
 
-If the user is taken to a page in the browser, the merchant has an identified user, can control the information given, and can ask the user to do further actions to continue the process, if relevant. For example, this can be used to welcome a user to the customer club, ask for more information, ask for consent to marketing, present terms of conditions for a subscription/purchase, request consent allowing the user to pay with Vipps.
+Alternatively, the merchant may redirect the user to a webpage in their browser. As the merchant will have an identified user
+entering their web page and as such can control the information given, and ask the user to do further actions to continue the process,
+if relevant. For example, this can be used to welcome a user to the customer club, ask for more information or preferences, or show offers or information of relevance.
```

Comment:
```suggestion
```suggestion
Alternatively, the merchant may redirect the user to it's website. The merchant will then have a identified user. This flow can be used to welcome a user to the customer club, ask for more information or preferences, or show offers or information of relevance.